conf/layer.conf

```diff
@@ -17,4 +17,4 @@ BBFILE_PRIORITY_webkit = "7"
 BB_DANGLINGAPPENDS_WARNONLY = "true"
 
 # Support the current actively  maintained Yocto releases
-LAYERSERIES_COMPAT_webkit = "dunfell gatesgarth hardknott"
+LAYERSERIES_COMPAT_webkit = "honister"
```

Comment:
Mmm, so this would break meta-webkit master compatibility with the Yocto releases previous to honister, far from ideal.
IMHO this is a bit unfortunate, but seems we have no choice.
I wonder if you know what is the reason behind this breaking change in bitbake to change the syntax for the overrides to use colons instead of underscores? I hope there is a good reason other than simply syntax sugar. I tried to search for that on the mailing list, but I didn't found anything explaining this change.

Reply:
this has been an imminent change  that was required for better bitbake functioning. There is lot of discussion on openembedded-architecture list https://lists.openembedded.org/g/openembedded-architecture/topic/should_we_change_variable/84225642?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,84225642

Reply:
we might be able to keep dunfell  onwards working, can you try this pull without this patch on latest dunfell and see if it works. Then we dont have to apply this one patch

Reply:
This change is sad but it is the only way to go right now but, before this, we need to create the branch from main to hardknott. The hardknott branch will keep the support for: dunfell, gatesgarth and itself. Since that moment I think we are going to do a double effort if we want to keep the recipes up to date for the latest LTS released (`dunfell`) until April 2022 (estimated release date of `kirkstone`).

Reply:
BTW @kraj  thanks for work on this.

Reply:
This change also implies a clean-up of the https://github.com/Igalia/meta-webkit/blob/main/.gitlab-ci.yml file.

Reply:
these changes will work with dunfell too. so we can keep the backward compatibility

Reply:
I have pushed changes to keep older releases in mix. Please test it on dunfell and see if all is ok. and then apply it.

Reply:
How this works? How are dunfell and older releases are able to parse the new syntax?

Reply:
Needed parser changes are cherry-picked into maintained releases already see [1] and [2]

[1] https://git.openembedded.org/bitbake/commit/?h=1.46&id=a6d5fb7554e3cf071e453db56a1e7469ac44277c
[2] https://git.openembedded.org/bitbake/commit/?h=1.46&id=c5418eae56cc50dbae7951c055434a0c456c53a4

Reply:
Thanks @kraj. I can confirm that. Patches are already in:
* Dunfell: https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/bitbake/lib?h=dunfell&id=1f7b04355c032cd0de4cbe9352745920964c8057
* Hardnknott: https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/bitbake/lib?h=hardknott&id=666d3421389cc3c01c56e92199cc32bddf1922de
*but also in the already EOL gatesgarth: https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/bitbake/lib?h=gatesgarth&id=f2d2136dbb3730abb87699ce03dd54c27ebb7833